raster-foundry/raster-foundry
# Fix legacy filter pane

## Overview
Fix legacy filter pane styling and missing AOI filter

### Checklist

- ~Styleguide updated, if necessary~
- ~Swagger specification updated~
- ~Symlinks from new migrations present or corrected for any new migrations~
- [x] Any content changes are properly templated using `BUILDCONFIG.APP_NAME` (Made sure styles work in the high-contrast view)
- ~Any new SQL strings have tests~

### Demo
![image](https://user-images.githubusercontent.com/4392704/55745856-3959c980-5a06-11e9-9857-d28c135564f5.png)

## Testing Instructions

- Verify that the AOI filter works as expected in the old filter view
- Verify that the styles now match the new filter pane (for the most part) and there isn't any invisible text / funky colors
- Verify the above in the high-contrast theme by setting the theme in your overrides.js file
Closes #4785
